board_client-sim/main.py

```
@@ -0,0 +1,96 @@
+# Patch sys.path so imports work
+import os
+sys.path.insert(0, os.path.abspath("."))
+
+# Set environment variable for model selection
+os.environ["UNICORN_SIM_MODEL"] = args.model
+
+# Import the simulation hardware layer
+from sim.hardware_sim import graphics, gu, set_brightness, WIDTH, HEIGHT, MODEL
+
+# Patch uw.hardware to point to our sim hardware
+import uw
```

Ruff (0.11.9)

`uw` imported but unused

Remove unused import: `uw`

(F401)